CHANGELOG.md

```
@@ -1,5 +1,21 @@
+---
+- RELEASE: 0.0.2正式版本发布
+- CHANGE: 优化部分错误处理
+- CHANGE&FIX: 修正文档错误
+
+25w02a
+---
+- PRE-RELEASE: 作为0.0.2的预发布版本, 请勿用于生产环境
+- CHANGE&FIX: 修正文档错误
+- CHANGE: 优化部分错误处理
+
+0.0.1
```

Markdownlint (0.37.0)

Expected: atx; Actual: setext
Heading style

(MD003, heading-style)